src/hooks/useTasks.js

```diff
@@ -52,23 +57,26 @@ function useOpenTasks(tasks, now, courtSettings) {
 
     for (let i = 0; i < incompleteTasks.length; i++) {
       const currentPhase = incompleteTasks[i].phase
-      const nextTransition = incompleteTasks[i].nextTransition
+      const nextTransition = getTermEndTime(
+        incompleteTasks[i].phaseEndTerm,
+        courtConfig
+      )
 
       if (
-        currentPhase !== DisputesTypes.Phase.AppealRuling &&
-        currentPhase !== DisputesTypes.Phase.ConfirmAppeal
+        currentPhase !== DisputePhase.AppealRuling &&
+        currentPhase !== DisputePhase.ConfirmAppeal
       ) {
         for (let j = 0; j < incompleteTasks[i].jurors.length; j++) {
           if (isVotingTaskOpen(incompleteTasks[i].jurors[j], currentPhase)) {
             openTasks.push({
               number: incompleteTasks[i].number,
               state: incompleteTasks[i].state,
-              createdAt: parseInt(incompleteTasks[i].createdAt, 10) * 1000,
+              createdAt: incompleteTasks[i].createdAt,
```

Comment:
Just wanted to sanity check that this was correct, since it didn't look like we changed `createdAt` in this PR.

Was this changed previously and we just forgot to update it here?

Reply:
Ah yeah i moved the tasks data transformation to the `subscription-hooks https://github.com/aragon/court-dashboard/pull/364/files#r436175026